contracts/adapters/lp/balancer/AbstractPoolAdapter.sol

```diff
@@ -8,13 +8,18 @@ import "../../../ILiquidityPosition.sol";
 import "../../../helpers/balancer/Interop.sol";
 
 abstract contract AbstractPoolAdapter is ILiquidityPosition, FactoryFriendly {
+    uint256 private constant DEFAULT_PARITY_TOLERANCE = 20;
+    uint256 private constant DEFAULT_SLIPPAGE = 50;
+    uint256 private constant DEFAULT_BLOCK_AGE = 5;
```

Comment:
These constants seem a little redundant. Why not just set the variables to these values when they are instantiated?

```solidity
    uint256 public parityTolerance = 20; // default to 20 basis points.
    uint256 public minBlockAge = 5; // default to 5 blocks.
    uint256 public slippage = 50; // default to 50 basis points.
```

I naively assume this will save a little gas on setup, since it's three less writes to storage.

If we are going to set them on on setup, then we should require the user to pass these values in as parameters. In which case the constants are still redundant.